### PR TITLE
Update to org.freedesktop.Sdk//20.08

### PR DESCRIPTION
**Pull request by youssefhabri**

### CODE REVIEW

**Comment by bilelmoussaoui** [comment]
Have you got a chance to try it ?

**Comment by youssefhabri** [comment]
Yep, and it seems to work just fine (played for around 30mins/15 rounds).

I wouldn't mind someone else trying it to confirm that it works.

**Comment by lionirdeadman** [comment]
bot, build

File: com.github.ryanakca.slingshot.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
     "app-id": "com.github.ryanakca.slingshot",
     "runtime": "org.freedesktop.Platform",
-    "runtime-version": "1.6",
+    "runtime-version": "19.08",
```
Comment (lionirdeadman) [inline com.github.ryanakca.slingshot.json]:
```suggestion
    "runtime-version": "20.08",
```

**Comment by hfiguiere** [comment]
Thanks a lot !

**Pull request merged**